lib/glossarist/concept.rb

```diff
@@ -15,8 +15,42 @@ class Concept < Model
     # @return [Hash<String, LocalizedConcept>]
     attr_reader :localizations
 
+    def deprecated_concepts
```

Comment:
1. Can we have a simple `related` concept relation?
2. For these `def deprecated_concepts`, the `deprecated` part is clearly derived from the relation name. Can we generate these methods? We should use the same verb form, i.e. `deprecates_concepts` for `:deprecates`

Reply:
> 1. Can we have a simple related concept relation?

We do have a simple `related concept` relation.

> For these `def deprecated_concepts`, the `deprecated` part is clearly derived from the relation name. Can we generate these methods? We should use the same verb form, i.e. `deprecates_concepts` for `:deprecates`

From what I understand you want to change the method names to `<relation_type>_concepts` i.e `deprecates_concepts`, `supersedes_concepts`, `compare_concepts` etc.

@ronaldtse did I get this right?

Reply:
Yes, meta programming would make things cleaner.